NEWS.md

```diff
@@ -5,6 +5,15 @@ Python 3.6 support dropped
 
   afew stopped supporting the older python version 3.6.
 
+Handle DMARC report with empty spf or dkim XML nodes
+
+DMARC Filter: allow to define subject regexp
+
+Get notmuch database path using Database wrapper
+
+Fix typo in DKIM filter docs
+
+HeaderMatchingFilter: do not convert user supplied tags
```

Comment:
why? This needs to contain some explanation, and again, it'd help looking at the PR description / changes.

Reply:
Yes I've added a more descriptive explanation of the change.